src/test/java/pages/GoogleHomePage.java

```diff
@@ -0,0 +1,28 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleHomePage extends BasePage {
+
+    @FindBy(name = "q")
+    private WebElement googleSearchField;
+
+    @FindBy(name = "btnK")
+    private WebElement googleSearchButton;
+
+    public GoogleHomePage(WebDriver driver) {
+        super(driver);
+
+        PageFactory.initElements(driver, this);
```

Comment:
declare this line in Base page and then you'll not need to declare it in other classes.